launch

```diff
@@ -62,6 +62,9 @@ CONFIG = {
             'rpi3': {
                 'kernel': 'sys/mimiker.img.gz',
             },
+            'pc': {
+                'kernel': 'sys/mimiker.elf',
+            },
```

Comment:
No, I'm not, but at some point the script checks the `config.kernel` value and I've decided to simply include the above line instead of changing the inner body of some function (in fact, I do use the `mimiker.elf` as a kernel image).

Reply:
Now it's used.